brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-885 Improvement : Removed Alignment option from toolbar for all block.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-885

### Screenshots

- https://share.getcloudapp.com/X6uRNNmN

### Types of changes




- Improvement

### How has this been tested?

- Drag and drop all blocks from Spectra.
- Check if any block has an alignment option in toolbar. We have removed the alignment option entirely from the toolbar for all blocks. 
- Alignment options are present on the right side inspector panel. Check Ss attached.

### Checklist:
